src/FactionMasterInvitationImprove.php

```diff
@@ -61,19 +61,22 @@ public function onLoad(): void {
 		$this->saveResource('tr_TR.yml');
 		$this->saveResource('pt_BR.yml');
 		$this->saveResource('ru_RU.yml');
+                $this->saveResource('es_SPA.yml');
 		$this->saveResource('config.yml');
 		ConfigUpdater::checkUpdate($this, $this->getConfig(), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "fr_FR.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "en_EN.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "tr_TR.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "pt_BR.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "ru_RU.yml", Config::YAML), "file-version", 1);
+                ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "es_SPA.yml", Config::YAML), "file-version", 1);
 		$this->LangConfig = [
 			"FR" => new Config($this->getDataFolder() . "fr_FR.yml", Config::YAML),
 			"TR" => new Config($this->getDataFolder() . "tr_TR.yml", Config::YAML),
 			"EN" => new Config($this->getDataFolder() . "en_EN.yml", Config::YAML),
 			"BR" => new Config($this->getDataFolder() . "pt_BR.yml", Config::YAML),
-			"RU" => new Config($this->getDataFolder() . "ru_RU.yml", Config::YAML)
+			"RU" => new Config($this->getDataFolder() . "ru_RU.yml", Config::YAML),
+                        "SPA" => new Config($this->getDataFolder() . "es_SPA.yml", Config::YAML),
```

Comment:
The comma at the end will not cause any syntax problem, but I think it would be better to remove it to avoid any confusion